conda_index/cli/__init__.py

```diff
@@ -91,6 +92,23 @@
         repodata_version=2 which is supported in conda 24.5.0 or later.
         """,
 )
+@click.option(
```

Comment:
We could replace this with an "add-only" or "no-remove" option, that would keep packages in the index even if they are not found in the filesystem.